hack/update-openapi.sh

```diff
@@ -35,6 +35,6 @@ GOPATH= ${OPENAPI_GEN} \
          --go-header-file ${SCRIPT_ROOT}/hack/empty.txt \
          ${verify}
 
-go build github.com/openshift/api/openapi/cmd/models-schema
+${GO:-go} build github.com/openshift/api/openapi/cmd/models-schema
```

Comment:
Earlier in this script (and for the other scripts) we have setup of the tooling variables such as
```
# Build openapi-gen when it's not present and not overriden for a specific file.
if [ -z "${OPENAPI_GEN:-}" ]; then
  ${TOOLS_MAKE} openapi-gen
  OPENAPI_GEN="${TOOLS_OUTPUT}/openapi-gen"
fi
```

I'm wondering if we should follow that pattern and have, then we can have a check that Go is actually installed as well, WDYT?
```
# Ensure the GO variable is set and that a go binary is installed
if [ -z "${GO:-}" ]; then
  if [ ! $(command -v go) ]; then
    echo Go is required to run this script but is not found on the system, please install Go and add it to the PATH
    exit 1
  fi
  # Might be nice to have a check go of the correct minimum version?
  GO=go
fi
```